src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+public class Ball {
+    private static int number;
+    private String color;
+
+    public Ball(String color) {
+        this.color = color;
+        Ball.number++;
```

Comment:
Ball should accept the color and number (black ball with 8 number).
In this case you just counting your balls

Reply:
done